### PR TITLE
Merge buildifier check into *-bazel rules

### PR DESCRIPTION
**Pull request by fejta**
Ensure skylark files are formatted correctly. Attempt to auto-fix any correctable lint errors.

### CODE REVIEW

**Comment by BenTheElder** [comment]
/assign
needs-rebase

File: hack/update-bazel.sh [hunk 1]
```diff
@@ -43,3 +44,6 @@ fi
 set -o xtrace
 "$gazelle" fix --external=external
 "$kazel" --cfg-path=./.kazelcfg.json
+find . -name BUILD -o -name BUILD.bazel -o -name '*.bzl' -type f \
+  \( -not -path '*/vendor/*' -prune \) \
```
Comment (BenTheElder) [inline hack/update-bazel.sh]: should we also buildifier BUILD files in vendor?

File: presubmit.sh [hunk 2]
```diff
@@ -45,6 +45,4 @@ export GO111MODULE=off # TODO(fejta): get rid of this
   set -o xtrace
   bazel test //... # This also builds everything
   ./verify/verify-boilerplate.sh --rootdir="$(pwd)" -v
-  ./verify/verify-bazel.sh
-  buildifier -mode=check $(find . -name BUILD -o -name '*.bzl' -type f -not -wholename '*/vendor/*')
```
Comment (fejta) [inline presubmit.sh]: The previous version didn't check vendor

File: hack/update-bazel.sh (hunk 1 above)
Comment (fejta) [reply hack/update-bazel.sh]: I don't think so? 

Either we're using the files downloaded from the repo -- in which case we shouldn't futz with the formatting) -- or we used gazelle to generate them.

Comment (BenTheElder) [reply hack/update-bazel.sh]: SGTM :+1:

**Pull request merged**